### PR TITLE
CP-49148: Clean py2 compatible code

### PR DESCRIPTION
**Pull request by liulinC**

### CODE REVIEW

**Review by bernhardkaindl — APPROVED** [review]

**Comment by liulinC** [comment]
ring3 BST: 202709 (Dev Run)
so far so good.

File: scripts/Makefile [hunk 1]
```diff
@@ -22,8 +19,6 @@ install:
 	mkdir -p $(DESTDIR)/usr/lib/systemd/system
 	mkdir -p $(DESTDIR)/usr/lib/yum-plugins
 	mkdir -p $(DESTDIR)$(OPTDIR)/packages/post-install-scripts
-	mkdir -p $(DESTDIR)$(SITE_DIR)
-	mkdir -p $(DESTDIR)$(SITE3_DIR)
```
Comment (edwintorok) [inline scripts/Makefile:26]: Are we sure this is not needed anymore for py3 either?

**Review by edwintorok — APPROVED** [review]
The changes remove conditional code around python2 support. Looks good (python2 compatibility already got dropped elsewhere).

**Comment by bernhardkaindl** [comment]
> ring3 BST: 202709 (Dev Run) so far so good.

Sounds good. This and the removed [qemu-vif-script](https://github.com/liulinC/xen-api/commit/27a373271fc973989027c46a6debae0f6ea22c04) (also included in the branch of this PR, but merged into feature/py3 already) hopefully working is/would be nice.

File: scripts/Makefile (hunk 1 above)
Comment (liulinC) [reply scripts/Makefile:26]: yes, it now goes to python3/Makefile.

Comment (bernhardkaindl) [reply scripts/Makefile:26]: Correct, these directories are now created (and used for installs) only in `python3/Makefile`.

Not removing these `mkdirs` in `scripts/Makefile` much earlier was an oversight.

**Pull request merged**